Providers/Scripts/3.x/Scripts/nxFileInventory.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```

Comment:
Were you able to make sure nxFileInventory, nxPackage and nxService work fine with python 3?

Reply:
yes nxFileInventory, nxPackage and nxService work with python3